Source/bmalloc/libpas/src/libpas/pas_fast_tls.h

```diff
@@ -84,20 +84,17 @@ struct pas_fast_tls {
         } \
     } while (false)
 
-#define PAS_FAST_TLS_GET(static_key, passed_fast_tls) ({ \
-        pas_fast_tls* fast_tls = (passed_fast_tls); \
-        void* result; \
-        if (fast_tls->is_initialized) \
-            result = pthread_getspecific((fast_tls)->key); \
-        else \
-            result = NULL; \
-        result; \
-    })
-
-#define PAS_FAST_TLS_SET(static_key, passed_fast_tls, value) do { \
+/* Regardless of whether the state of fast_tls, we can access static_key. */
```

Comment:
I suggesting adding a bit more detail to the comment e.g.
```
/* This is the PAS_HAVE_THREAD_KEYWORD case. Hence, static_key here is not actually a key, but a thread
   local cache of the value (declared with the __thread attribute). Regardless of whether fast_tls has been
   initialized yet or not, it is safe to access this thread local cache of the value. */
```
Am I understanding this correctly?

Reply:
Yes.